include/tvm/runtime/c_runtime_api.h

```diff
@@ -38,7 +38,8 @@
 #define TVM_DLL __declspec(dllimport)
 #endif
 #else
-#define TVM_DLL
+// The TVM_DLL tag cannot appear right after doxygen comments, else doxygen parser is confused.
+#define TVM_DLL __attribute__((visibility("default")))
```

Comment:
same comment like NNVM_DLL.

Reply:
Thought about using __GNUC__ >= 4 but in the case of __GNUC__ < 4 the macro will not be defined leading to symbols not exported so it will not work anyway. And __GNUC__ < 4 is way too old to be obsolete and nobody use it any more. So to keep code sanity let's just always define the macro for any GNU version.